src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -119,28 +178,103 @@ pub struct Policy<'a, B: CryptoOps> {
     /// The validation time. All certificates validated by this policy must
     /// be valid at this time.
     pub validation_time: asn1::DateTime,
+
+    /// An extended key usage that must appear in EEs validated by this policy.
+    pub extended_key_usage: ObjectIdentifier,
+
+    /// The set of permitted signature algorithms, identified by their
+    /// algorithm identifiers.
+    ///
+    /// If not `None`, all certificates validated by this policy MUST
+    /// have a signature algorithm in this set.
+    ///
+    /// If `None`, all signature algorithms are permitted.
+    pub permitted_algorithms: Option<HashSet<AlgorithmIdentifier<'a>>>,
+
+    pub(crate) critical_ca_extensions: HashSet<ObjectIdentifier>,
+    pub(crate) critical_ee_extensions: HashSet<ObjectIdentifier>,
 }
 
 impl<'a, B: CryptoOps> Policy<'a, B> {
-    /// Creates a new policy with the given `CryptoOps`, an optional subject,
-    /// and a validation time.
-    pub fn new(ops: B, subject: Option<Subject<'a>>, time: asn1::DateTime) -> Self {
+    /// Create a new policy with defaults for the certificate profile defined in
+    /// the CA/B Forum's Basic Requirements.
+    pub fn webpki(ops: B, subject: Option<Subject<'a>>, time: asn1::DateTime) -> Self {
```

Comment:
Why did WebPKI come back?

Reply:
Bad merge, will fix.

Reply:
Fixed in [90e502b](https://github.com/pyca/cryptography/pull/9642/commits/90e502b9e240b3c7aa4751d3b57f2c94f38297da) -- I kept the doc change since the defaults are changing internally, but I can also revert that if you'd prefer.